### PR TITLE
Add disclaimer if there are no store orders

### PR DESCRIPTION
**Pull request by TPGPL**

### CODE REVIEW

File: resources/lang/en/store.php [hunk 1]
```diff
@@ -89,6 +89,7 @@
         'paid_on' => 'Order placed :date',
 
         'invoice' => 'View Invoice',
+        'no_orders' => 'No orders to view.',
```
Comment (nanaya) [inline resources/lang/en/store.php]: I don't think `to view` part is needed.

Comment (TPGPL) [reply resources/lang/en/store.php]: i believe in most pages i visited, it was `No <item> to view.`

**Comment by nanaya** [comment]
You should first check if your changes actually work.

**Comment by TPGPL** [comment]
I would if docker actually worked

**Comment by nanaya** [comment]
Report problems on docker instead of blindly making changes and pull requests.

**Pull request merged**